### ISSUE TITLE
Initially image is loaded in max zoom level. Is it possible to load it in fit to screen.

### ISSUE THREAD

**Issue by bibinjacobpulickal** [dblock/ARTiledImageView#17]

**Comment by dblock** [comment]
I think [this](https://github.com/artsy/eigen/blob/505b08c70f58d703f19436bc41975b84b052643e/Artsy/View_Controllers/Fair/Utils/ARFairMapZoomManager.m#L42) does what you want?

**Comment by bibinjacobpulickal** [comment]
yes, it worked to some extend. But, still the image is not in fit to screen. Scrolling in one direction is still present.

**Comment by dblock** [comment]
Is this duplicate of #18?

**Comment by bibinjacobpulickal** [comment]
No, that is different. Here what I mean is, when an image is opened the whole image should be visible an should be able to start zooming from there. (Like when you open an image from gallery, should also be centered.)

**Comment by bibinjacobpulickal** [comment]
![simulator screen shot 7 oct 2017 14 59 02](https://user-images.githubusercontent.com/26866256/31306666-3771e5d8-ab72-11e7-8d3b-2f2c5f5cfe38.png)
![simulator screen shot 7 oct 2017 15 12 49](https://user-images.githubusercontent.com/26866256/31306667-3772568a-ab72-11e7-884c-2252f3731b42.png) 

I require 1 instead of 2.
